Controller/AuthorizationController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
```

Comment:
@ajgarlag Repository is about PHP7.2+. It would be great if you provide ```declare(strict_types=1)``` to every class and provide type for every argument in methods (sometimes it's missed). It seems it must be done for all project.

Reply:
@Derkien You are right, it must be done for all files in project, so I think it should be done in a different PR modifying the `.php_cs.dist` file to accomplish it. WDYT?

Reply:
@ajgarlag @Derkien Definitely in a separate PR so we don't go out of scope in this one.

Reply:
@ajgarlag Sounds like a plan :) 👍 
@spideyfusion Ofc in separate.